package.json

```diff
@@ -22,6 +22,7 @@
   ],
   "main": "app.js",
   "dependencies": {
+    "@snyk/protect": "latest",
```

Comment:
We should lock this down to a major version to ensure we don't have breaking changes introduced later.